packages/utils/src/omit.ts

```diff
@@ -1,9 +1,15 @@
 /**
- * Omit keys from object.
+ * The `omit` function creates a new object by omitting specified keys from the
+ * original object. It takes an object and an array of keys to exclude,
+ * returning a new object that contains only the properties not specified in
+ * the keys array. This is useful for filtering out sensitive or unnecessary
+ * information from an object before using it.
```

Comment:
**thought:** I don't really know if we _need_ to present verbally what the function does

I think the example perfectly shows this.

Reply:
Useful to have when consuming this information via the documentation site.